holmes/plugins/toolsets/oom_kill.py

```
@@ -0,0 +1,95 @@
+                "hold_seconds": ToolParameter(
+                    description=(
+                        "How long to keep the memory allocated before exiting. Defaults to 300 seconds."
+                    ),
+                    type="integer",
+                    required=False,
+                ),
+            },
+            toolset=toolset,
+        )
+
+    def _invoke(self, params: dict, context: ToolInvokeContext) -> StructuredToolResult:
```

Ruff (0.14.10)

Unused method argument: `context`

(ARG002)